tox.ini

```diff
@@ -5,18 +5,22 @@ envlist =
        py27-django1.{10,11}-drf3.{5,6,7,8}
        {py34,py35,py36}-django2.0-drf3.{7,8,9}
        {py35,py36}-django2.1-drf3.{8,9}
+       {py35,py36}-django2.1-drf3.{8,9}
+       {py37}-django3.0-drf3.{8,9}
```

Comment:
It looks like your build is failing cause drf just added Django 3.0 support in 3.11. Can you try adding `drf3.11: djangorestframework>=3.11,<3.12` below and changing this to `{py37}-django3.0-drf3.{11}`.

Looking forward to 3.0 compatibility.

Reply:
Thanks Carlos